tox.ini

```diff
@@ -80,10 +80,10 @@ commands_pre =
     # the rest is for good measure
     headless: pip uninstall -y pytest-qt qtpy pyqt5 pyside2
 commands =
-    !headless: python -m pytest {env:PYTEST_PATH:} --color=yes --basetemp={envtmpdir} \
+    !headless: python -m pytest napari --color=yes --basetemp={envtmpdir} \
         --cov-report=xml --cov={env:PYTEST_PATH:napari} --ignore tools --maxfail=5 \
-        --json-report --json-report-file={toxinidir}/report-{envname}.json {posargs}
-        --skip_examples
+        --json-report --json-report-file={toxinidir}/report-{envname}.json \
+        --skip_examples {posargs}
```

Comment:
Why are these changes here?

Reply:
because `--skip_examples` was ignored and the test executes also examples tests. This makes tests with tox fails local for me.

Reply:
Hey @Czaki, I noticed that i'm unable to use posargs anymore to restrict tox tests and tracked it down to this PR.  If I opened a PR to restore the `{env:PYTEST_PATH:}`... is it going to break something for you?  I'd like to be able to run tox without having to run the whole test suite but this breaks that